shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -678,8 +678,11 @@ void onDestroyView() {
       flutterView.getViewTreeObserver().removeOnPreDrawListener(activePreDrawListener);
       activePreDrawListener = null;
     }
-    flutterView.detachFromFlutterEngine();
-    flutterView.removeOnFirstFrameRenderedListener(flutterUiDisplayListener);
+
+    if (flutterView != null) {
```

Comment:
Can you add a shortened version of your explanation as a comment here? 
Perhaps something like 
```suggestion
    // flutterView can be null in instances where a delegate.onDestroyView is called without onCreateView being called. See https://github.com/flutter/engine/pull/41082 for more detail. 
    if (flutterView != null) {
```

Reply:
Your comment looks concise and pretty good to me so I'll take the liberty to use the same. 🙂